src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -307,6 +307,54 @@ def transform(
                 ),
             )
 
+            # sampling rate
+        elif transform_method.__name__ == 'resample':
+            resample_rate = kwargs.pop('resampling_rate')
+
+            if self.trial_columns is None:
+                self.frame = transforms.resample(
+                    frame=self.frame,
+                    resampling_rate=resample_rate,
+                    n_components=self.n_components,
+                    **kwargs,
+                )
+            else:
+                # Manipulate columns to exclude trial columns
+                resample_columns = kwargs.pop('columns', 'all')
```

Comment:
When triggering resampling by calling `GazeDataFrame.transform('resample', ...)` or `Dataset.apply('resample',...)` there is no default `columns=all` set. I need to check this attribute here to exclude the trial_columns from further processing in `resample`. Is `kwargs.pop('columns', 'all')` the preferred way to handle this here?